AttributeError on connection issues

It's expecting ```e.message``` to exist on any requests exception, but ```ConnectionError``` does not have that attribute:

```
    def _do_request(self, method, path, params=None, data=None):
        """Query Marathon server."""
        headers = {
            'Content-Type': 'application/json', 'Accept': 'application/json'}
        response = None
        servers = list(self.servers)
        while servers and response is None:
            server = servers.pop(0)
            url = ''.join([server.rstrip('/'), path])
            try:
                response = requests.request(
                    method, url, params=params, data=data, headers=headers,
                                            auth=self.auth, timeout=self.timeout)
                marathon.log.info('Got response from %s', server)
            except requests.exceptions.RequestException as e:
                marathon.log.error(
>                   'Error while calling %s: %s', url, e.message)
E               AttributeError: 'ConnectionError' object has no attribute 'message'
```

--- a/marathon/client.py
+++ b/marathon/client.py
@@ -65,7 +65,7 @@ class MarathonClient(object):
                 marathon.log.info('Got response from %s', server)
             except requests.exceptions.RequestException as e:
                 marathon.log.error(
-                    'Error while calling %s: %s', url, e.message)
+                    'Error while calling %s: %s', url, str(e))
 
         if response is None:
             raise MarathonError('No remaining Marathon servers to try')

